packages/vscode-extension/README.md

```diff
@@ -76,15 +75,14 @@ At its core, the Teams app embraces three components:
 
 When an app is installed, the Teams client parses the manifest file to determine needed information like the name of your app and the URL where the services are located.
 
-- To configure your app, navigate to the `appPackage/manifest.source.json` file and edit the manifest.
-- To validate your manifest file, click `Validate manifest file` in the sidebar menu.
+- To configure your app, navigate to the `appPackage/manifest.template.json` file or from UI to select `DEVELOPMENT`-`Edit manifest file` and edit the manifest.
 - The toolkit will automatically update the app registration data accordingly during app side-loading and publish.
 
 ### Preview your app on your local/remote dev environment
 
```

Comment:
In each section, better to have the link to the related contents in docs site. This one can be https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/debug-local

Reply:
good point, added the read more doc link at end of each section.